.github/workflows/ci.yml

```diff
@@ -284,6 +284,57 @@ jobs:
           cache-from: type=registry,ref=ghcr.io/fuellabs/fuel-core-build-cache:latest
           cache-to: type=registry,ref=ghcr.io/fuellabs/fuel-core-build-cache:latest,mode=max
 
+  # duplicate of publish-docker-image, but with profiling features enabled
+  # this is split into a separate action since it takes longer to build
+  publish-docker-image-profiling:
```

Comment:
I guess we only want to do that during release?

Reply:
Following from continuous delivery best practices, every commit should be deployable